lighthouse-core/report/html/templates.html

```diff
@@ -783,24 +783,64 @@
 
 <template id="tmpl-lh-3p-filter">
   <style>
+    /* The input checkbox element is invisible by should span the same width and height
```

Comment:
I modeled this CSS off of how the metric description pill works.

This would be a great candidate for a custom web component.

Reply:
```suggestion
    /* The input checkbox element is invisible but should span the same width and height
```